modules/vcf2maf/main.nf

```diff
@@ -0,0 +1,62 @@
+def VERSION = '1.6.21'
+
+process VCF2MAF {
+    /*
+    This runs vcf2maf on an uncompressed VCF file and converts it to MAF format.
+
+    If VEP is installed you will need to add the VEP cache as a path. If you wish to skip VEP, add `--inhibit-vep` to ext.args. Running VEP requires a VEP cache to be present.
+
+    It is recommended to set the --species and --ncbi-build in ext.args (use the module config). It may also be necessary to set --tumor-id and --normal-id for correct parsing of the VCF.
+    */
+
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::vcf2maf=1.6.21 bioconda::ensembl-vep=106.1" : null)
```

Comment:
@maxulysse how is there another update already 😱

Reply:
I know...